Project-Euler/test/Problem015.test.js

```diff
@@ -0,0 +1,32 @@
+import { latticePath } from '../Problem015.js'
+
+describe('Lattice Paths', () => {
+  test('if grid size is 2', () => {
```

Comment:
Such repetitive tests should use `test.each` / `it.each` to reduce the boilerplate.